src/components/download/DataDownloadDisplayComponent.vue

```diff
@@ -185,18 +189,35 @@ const filterId = props.topologyNode.filterIds
   ? props.topologyNode.filterIds[0]
   : undefined
 const allLocations = ref<Location[]>([])
+const allParameters = ref<TimeSeriesParameter[]>([])
 const locations = ref<Location[]>([])
 const selectedLocations = ref<Location[]>([])
 
-const parameters = ref<TimeSeriesParameter[]>([])
-const selectedParameters = ref<TimeSeriesParameter[]>([])
+const parameterQualifiers = ref<ParameterQualifiersHeader[]>([])
+const selectedParameterQualifiers = ref<ParameterQualifiersHeader[]>([])
 
 allLocations.value = await getLocations()
 locations.value = allLocations.value
 selectedLocations.value = allLocations.value
 
-parameters.value = await getParameters()
-selectedParameters.value = parameters.value
+const timeSeriesResults = await getTimeSeriesHeaders()
```

Comment:
Top-level `await` blocks setting up the component until the request finishes, is this what we want?

In any case, since the fetches are independent, they can be executed concurrently with `Promise.all` or `Promise.allSettled`.